src/org/labkey/test/Locator.java

```diff
@@ -841,11 +841,6 @@ public static XPathLocator gwtCheckBoxOnImportGridByColLabel(String label)
         return tagWithAttribute("input", "type", "checkbox").withPredicate(xpath("../../following-sibling::td/span").containing(label));
     }
 
-    public static XPathLocator gwtNextButtonOnImportGridByColLabel(String label)
-    {
-        return tagWithClass("div", "x-tbar-page-next").withPredicate(xpath("../preceding-sibling::td/span").containing(label));
-    }
-
     public static Locator permissionRendered()
```

Comment:
Can we get rid of some of these other gwt locators?

Reply:
Done. I got rid of any used gwt locators in this file.